app/subscriber/src/features/hooks/useSubMediaGroups.ts

```diff
@@ -26,6 +26,16 @@ export const useSubMediaGroups = (
 } => {
   const [subMediaGroups, setSubMediaGroups] = useState<ISubMediaGroupItem[]>([]);
   const [{ userInfo }] = useApp();
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  const MediaTypeAllSources = [
```

Comment:
This will break once the media types are changed with the Editor admin.  Can we get this list from the lookups?

Reply:
We are creating mediaTypes in DB using  [01-MediaType.sql migration](https://github.com/bcgov/tno/blob/dev/libs/net/dal/Migrations/1.0.114/Up/PostUp/01-MediaType.sql) and creating mapping using [00-MediaTypeSearchMapping.sql](https://github.com/bcgov/tno/blob/dev/libs/net/dal/Migrations/1.0.95/Up/PostUp/00-MediaTypeSearchMapping.sql) 

Then, we are retrieving all mediaTypes and sources using [useLookup](https://github.com/bcgov/tno/blob/dev/app/subscriber/src/features/filter-media/FilterMediaLanding.tsx#L22). This call gives us all records for mediaTypes and sources. Then, [useSubmBediaGroups](https://github.com/bcgov/tno/blob/dev/app/subscriber/src/features/filter-media/FilterMediaLanding.tsx#L23) hook takes this as input to assign which sources are mapped to specific MediaType.

The only way I see the mediaType can be modified is through another migration as I can see. I checked in the editor side and we do not have the option to modify a mediaType. 

Not sure what I am missing.

Reply:
https://dev.editor.mmi.gov.bc.ca/admin/media-types

Reply:
Also each source can be mapped to a media types through the source admin page. https://dev.editor.mmi.gov.bc.ca/admin/sources/261

Reply:
And the Series can be mapped to media types through the show/program admin page.  https://dev.editor.mmi.gov.bc.ca/admin/programs/788

Reply:
Added the Settings configuration for MediaTypes that do not follow the mapping and query just by Media Type.  

Ready for your review.